lib/kitchen/directions/bake_chapter_key_concepts.rb

```diff
@@ -0,0 +1,59 @@
+# frozen_string_literal: true
+
+module Kitchen
+  module Directions
+    # Bake directions for eoc key concepts
+    #
+    module BakeChapterKeyConcepts
+      def self.v1(chapter:, metadata_source:)
+        metadata_elements = metadata_source.children_to_keep.copy
+
+        fix_titles = chapter.pages.map do |page|
+          key_concepts = page.key_concepts
+          next if key_concepts.none?
+
+          key_concepts.search('h3').trash
+          key_concepts.map do |key_concept|
+            id = key_concept.id.split('fs-id')
+            key_concept.prepend(child:
+              <<~HTML
+                <a href="##{id[0]}0">
+                  <h3 data-type="document-title" id="#{id[0]}0">
+                    <span class="os-number">#{chapter.count_in(:book)}.#{page.count_in(:chapter) - 1}</span>
```

Comment:
The `- 1` leads to numbers like `1.0` which I see in the spec.  But baked Calculus always has key concepts starting at `x.1`, not `x.0`.  

I think this is because the book you are working on has introduction pages that are uncounted at the start of each chapter.  This code may only be used in books with introductions in each chapter, but it may not be.  

You can [see in the Chemistry recipe that we get around this by only iterating over the non-introduction pages](https://github.com/openstax/recipes/blob/main/books/chemistry/bake#L62-L70), which ensures that our `page.count_in(:chapter)` always starts at 1 for the first non-introductory page.   You can do the same thing up on line 11: `chapter.pages('$:not(.introduction)')` and then get rid of the `- 1`.

The fact that this crops up in at least one place in a recipe and in two other directions leads me to think that we should add a `non_introduction_pages` method to `ElementEnumeratorBase` [here](https://github.com/openstax/kitchen/blob/main/lib/kitchen/element_enumerator_base.rb#L53) and also a delegator method from `ElementBase` [here](https://github.com/openstax/kitchen/blob/main/lib/kitchen/element_base.rb#L597).  Then you can say `chapter.non_introduction_pages` here and the other places.